test/route.7.test.js

```diff
@@ -283,15 +279,15 @@ test('GET route with body schema should throw', t => {
         reply.send({ hello: 'world' })
       }
     })
-  }, new Error('Body validation schema for GET:/get route is not supported!'))
+  }, createError('FST_ERR_ROUTE_BODY_VALIDATION_SCHEMA_NOT_SUPPORTED', 'Body validation schema for GET:/get route is not supported!')())
```

Comment:
I think the change to `createError` instead of the existing `new Error` in these tests is to match more of the thrown error than just the message. Is that correct? If not, what is the reasoning?

Reply:
Yes thats right, the node `assert.throws` is more strict and does not pass with a generic `Error` with message as before, because it expects `FastifyError` with the given name, code (and other fields) instead. 

I haven't found a more elegant way to pass this assertion.

Reply:
👍